src/django_upgrade/fixers/use_l10n.py

```diff
@@ -0,0 +1,44 @@
+"""
+USE_L10N setting is deprecated:
+https://docs.djangoproject.com/en/4.0/releases/4.0/#localization
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable
+
+from tokenize_rt import Offset, Token
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer, State, TokenFunc
+from django_upgrade.tokens import LOGICAL_NEWLINE, find
+
+fixer = Fixer(
+    __name__,
+    min_version=(4, 0),
+)
+
+
+@fixer.register(ast.Assign)
+def visit_Assign(
+    state: State,
+    node: ast.Assign,
+    parent: ast.AST,
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        len(node.targets) == 1
+        and isinstance(node.targets[0], ast.Name)
+        and node.targets[0].id == "USE_L10N"
+        and isinstance(node.value, ast.Constant)
+        and node.value.value is True
+        and state.looks_like_settings_file()
+    ):
+        yield ast_start_offset(node), partial(remove_assignment, node=node)
+
+
+# TODO: copied from default_app_config.py - should we move this to a common
```

Comment:
Question here

Reply:
Yeah, put it in `tokens`, under the `# Rewriting functions` header. i think it could also be renamed to `remove_logical_line` ?

It also has a mistake to unod: it takes `node` but doesn't use it, so you can drop the arg.

Reply:
We already had `erase_node` which works equivalent, so I've updated both fixers to use that.